lib/features/email/domain/model/email_print.dart

```diff
@@ -46,26 +46,28 @@ class EmailPrint with EquatableMixin {
   });
 
   factory EmailPrint.generate({
-    required PrintEmailAction printEmailAction,
+    required BuildContext context,
```

Comment:
I don't think we should pass framework's object to model layer. Instead, try to declare each String accordingly. E.g. If we need a String for appName, add:
```dart
required String appName
```

Reply:
good detect. done